src/commercetools/testing/utils.py

```diff
@@ -113,6 +114,38 @@ def updater(self, obj, action):
     return updater
 
 
+def update_datetime_attribute(dst: str, src: str):
+    def updater(self, obj, action):
+        value = getattr(action, src)
+
+        if not isinstance(value, datetime):
+            raise TypeError(f"Unsupported datetime object type: f{type(value)}")
+
+        if obj.get(dst) != value:
+            new = copy.deepcopy(obj)
+            new[dst] = value.isoformat()
+            return new
+        return obj
+
+    return updater
+
+
+def update_nested_object_attribute(dst: str, src: str):
+    def updater(self, obj, action):
+        values = getattr(action, src)
+
+        if not isinstance(values, typing.List):
```

Comment:
This might work, but we should not use typing stuff for isinstance, use `list` instead.